src/components/SearchError.tsx

```diff
@@ -0,0 +1,47 @@
+import {View} from 'react-native'
+
+import {atoms as a, useBreakpoints} from '#/alf'
+import * as Layout from '#/components/Layout'
+import {Text} from '#/components/Typography'
+import {Cross} from './icons/Cross'
+
+export function SearchError({
+  title,
+  children,
+}: {
+  title?: string
+  children?: React.ReactNode
+}) {
+  const {gtMobile} = useBreakpoints()
+
+  // NOTE (chenyu): asking Hannah about the new fill color for the SVG
+  return (
+    <Layout.Content>
+      <View
+        style={[
+          a.align_center,
+          a.gap_4xl,
+          {
+            paddingHorizontal: 10,
+            paddingVertical: 150,
+          },
+        ]}>
+        <Cross width={32} fill={'#D1D8E0'} />
+        <View
+          style={[
+            a.flex,
+            a.flex_col,
+            a.align_center,
+            {maxWidth: gtMobile ? 394 : 294},
```

Comment:
Same comment about values here as I made above, how do we know `394` and `294` are values we want to work with? I'm sure they satisfy the requirement of fixing this issue/UI

cc: @estrattonbailey

Reply:
Same above

Reply:
Darrin mentioned since this is a temp solution, this should be fine